MSAL/src/MSALPublicClientApplication.m

```diff
@@ -1471,7 +1471,7 @@ - (void)getDeviceInformationWithParameters:(MSALParameters *)parameters
         }
         else
         {
-            MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, nil, @"Retrieved device info %@", deviceInformation);
+            MSID_LOG_WITH_CTX_PII(MSIDLogLevelInfo, nil, @"Retrieved device info %@", MSID_PII_LOG_MASKABLE(deviceInformation));
```

Comment:
Just curious, if the deviceInfo already hased, is the output PII or regular info?

Reply:
My understanding is if the value is array, it will only show the length of array, if error, will because error domain + error code. For the rest, will show as null/nonull
 
So if it is hashed as a string, it will only show as nonull/null

Let me know if that is correct